prospero-cli/src/main/java/org/wildfly/prospero/cli/ExecutionExceptionHandler.java

```diff
@@ -0,0 +1,39 @@
+package org.wildfly.prospero.cli;
```

Comment:
Can you add the license header please?

Reply:
+1 will update my idea settings finally and add the headers in a new commit.

Reply:
I created new PR where copyright headers are added in all java files: https://github.com/wildfly-extras/prospero/pull/106